.github/workflows/tests.yml

```
@@ -51,7 +51,7 @@ jobs:
         run: |
           docker build -t brew .
           docker create \
-            -e GITHUB_ACTIONS -e GITHUB_BASE_REF -e GITHUB_REF -e GITHUB_REPOSITORY -e GITHUB_SHA \
+            -e GITHUB_ACTIONS -e GITHUB_BASE_REF -e GITHUB_REF -e GITHUB_OUTPUT -e GITHUB_REPOSITORY -e GITHUB_SHA \
```

Currently these environment variables are sorted alphabetically. I suggest the following change, to keep it that way.

```suggestion
            -e GITHUB_ACTIONS -e GITHUB_BASE_REF -e GITHUB_OUTPUT -e GITHUB_REF -e GITHUB_REPOSITORY -e GITHUB_SHA \
```